config/gulp/build.js

```diff
@@ -38,6 +41,40 @@ gulp.task('clean-assets', function (done) {
   );
 });
 
+// We might be using USWDS from a git repository instead of npm, in
+// which case it might not have the pre-built assets we need. If that's
+// the case, we'll want to build those assets.
+gulp.task('build-uswds-if-needed', function () {
+  const rootDir = path.normalize(path.join(__dirname, '..', '..'));
+  const uswdsDir = path.join(rootDir, 'node_modules', 'uswds');
+  const fractalIndex = path.join(uswdsDir, 'build', 'index.html');
+
+  if (fs.existsSync(fractalIndex)) {
+    dutil.logMessage('build-uswds-if-needed', 'USWDS is already built.');
+    return Promise.resolve();
+  } else {
+    dutil.logMessage('build-uswds-if-needed', 'Building USWDS...');
+    return new Promise((resolve, reject) => {
+      child_process.spawn(
+          'npm',
+          [ 'run', 'federalist' ],
```

Comment:
I'm confused by this. The `web-design-standards` repo doesn't have a `federalist` script, only this docs repo does. Should this be `npm run build` from that directory?